### ISSUE TITLE
Can you please add types?

### ISSUE THREAD

**Issue by lumberman** [react-native-hooks/keyboard#13]
Amazing library. Can you please add types support?

**Comment by wodka** [comment]
I use a keyboard.d.ts file containing:

```typescript
declare module '@rnhooks/keyboard' {
  const hook: () => [boolean, () => any]
  export default hook
}
```

**Comment by divyanshu013** [comment]
I haven't tried adding types to a JS library before. @wodka would you like to create a PR for this if you're interested? I'm wondering if adding a `index.d.ts` file to the root will suffice.

**Comment by hannojg** [comment]
added type support in #21

**Comment by divyanshu013** [comment]
This is released in v1.1.0
